dotcom-rendering/src/components/ArticlePage.tsx

```diff
@@ -62,12 +67,18 @@ export const ArticlePage = (props: WebProps | AppProps) => {
 					:root {
 						/* Light palette is default on all platforms */
 						${paletteDeclarations(format, 'light')}
-
+						body {
+							color: ${sourcePalette.neutral[7]};
+						}
```

Comment:
This is already defined in the css resets provided by libs. However, we're adding this here to make the light / dark coupling more explicit. @mxdvl